src/widgets/courses/ui/courses.tsx

```diff
@@ -1,14 +1,21 @@
+import classNames from 'classnames/bind';
 import { COURSE_STALE_AFTER_DAYS } from '@/app/const';
 import { type Course, CourseCard } from '@/entities/course';
 import { getActualData } from '@/shared/helpers/getActualData';
 import { isCourse } from '@/shared/helpers/is-course';
 import { useDataByName } from '@/shared/hooks/use-data-by-name';
 import { WidgetTitle } from '@/shared/ui/widget-title';
+import { DataMap } from 'data';
 
-import './courses.scss';
+import styles from './courses.module.scss';
+
+const cx = classNames.bind(styles);
+
+type PathNames = keyof DataMap;
+const PATH: PathNames = 'courses';
```

Comment:
Let's move this to `courses/constants.ts`

Reply:
Done